src/server/capacity_unit_calculator.cpp

```diff
@@ -60,25 +88,32 @@ int64_t capacity_unit_calculator::add_write_cu(int64_t write_data_size)
     return write_cu;
 }
 
-void capacity_unit_calculator::add_get_cu(int32_t status, const dsn::blob &value)
+void capacity_unit_calculator::add_get_cu(int32_t status,
+                                          const dsn::blob &key,
+                                          const dsn::blob &value)
 {
+    _pfc_get_bytes->add(key.size() + value.size());
     if (status != rocksdb::Status::kOk && status != rocksdb::Status::kNotFound) {
         return;
     }
     add_read_cu(value.size());
 }
 
 void capacity_unit_calculator::add_multi_get_cu(int32_t status,
+                                                const dsn::blob &hash_key,
                                                 const std::vector<::dsn::apps::key_value> &kvs)
 {
-    if (status != rocksdb::Status::kOk && status != rocksdb::Status::kNotFound &&
-        status != rocksdb::Status::kIncomplete && status != rocksdb::Status::kInvalidArgument) {
-        return;
-    }
     int64_t data_size = 0;
     for (const auto &kv : kvs) {
         data_size += kv.key.size() + kv.value.size();
     }
+    int64_t multi_get_bytes = hash_key.size() + data_size;
+    _pfc_multi_get_bytes->add(multi_get_bytes);
+
+    if (status != rocksdb::Status::kOk && status != rocksdb::Status::kNotFound &&
```

Comment:
Why these four status don't need add read cu?

Reply:
1. no success, so no cu, this is previous rule, this pr don't change cu rule.
2. no success, but still has throughput. 

about cu, you can see this pr https://github.com/XiaoMi/pegasus/pull/504, especially the aliyun link in description.